### DISCUSSION TITLE
How to connect to Bike via Terminal

### DISCUSSION THREAD

**Discussion by chiquitakool** [cagnulein/qdomyos-zwift#282] [Q&A]
Hi,

I purchased a raspberry pi zero w. installed via recommended way.

running `sudo ./qdomyos-zwift -no-gui` only searches endlessly for my bike, with -name option and name of the bluetooth device e.g. `sudo ./qdomyos-zwift -no-gui -name i-console+1764` or 'i-console+1764' it also won't stop searching. Tested with bike already on prior boot and vice versa. Android app is directly connecting to the bike.

How to connect ? 😄

**Comment by cagnulein** [comment]
because you need to specify the toorx bike setting available only with the
-qml parameter (remove the -no-gui)
let me know

Roberto Viola
Software engineer and open source enthusiast
http://robertoviola.cloud



Il giorno mer 21 apr 2021 alle ore 15:51 fab0nymous <
***@***.***> ha scritto:

**Reply by chiquitakool** [reply]
Ok thanks 
`pi@raspberrypi:~/qdomyos-zwift/src $ sudo ./qdomyos-zwift -qml
OK, you are root.
qt.qpa.screen: QXcbConnection: Could not connect to display 
Could not connect to any X display.`

Guess I have to upgrade my lite version of raspbian..

**Comment by cagnulein** [comment]
yeah you have to run with a graphical display. also check
https://github.com/cagnulein/qdomyos-zwift/issues/193#issuecomment-795685369

Roberto Viola
Software engineer and open source enthusiast
http://robertoviola.cloud



Il giorno mer 21 apr 2021 alle ore 16:03 fab0nymous <
***@***.***> ha scritto:
